tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -125,6 +125,12 @@ bgp/test_bgp_bbr.py:
       - "'t1' not in topo_type"
       - "platform in ['x86_64-8111_32eh_o-r0']"
 
+bgp/test_bgp_config.py:
+  skip:
+    reason: "Only supported on Cisco 8111 T1(compute ai) platform."
+    conditions:
+      - "platform not in ['x86_64-8111_32eh_o-r0']"
```

Comment:
We need this on the other AI BE platform as well, such as 8122.Can we use other signature to detect if we need to run this case or not?

Reply:
Currently MSFT internal testbeds are NOT marked as "backend". As far as I can see, there is no such signature to detect if we need run this testcase.

Reply:
Ok, got it. then we should list all the existing platforms here as well. 8122, th5, 5600